code/modules/mob/camera/eye.dm

```diff
@@ -0,0 +1,150 @@
+/mob/camera/eye
+	name = "Inactive Camera Eye"
+	icon = 'icons/obj/abductor.dmi'
+	icon_state = "camera_target"
+	alpha = 127
+	invisibility = SEE_INVISIBLE_OBSERVER
+
+	var/list/visibleCameraChunks = list()
+	var/mob/living/user = null
+	var/user_previous_remote_control = null
+	var/origin = null
+	var/relay_speech = FALSE
+	var/static_visibility_range = 16
+	// Decides if it is shown by AI Detector or not
+	var/ai_detector_visible = TRUE
+	var/visible_icon = FALSE
+	var/list/networks = list("SS13")
+	var/image/user_image = null
+
+	// Camera acceleration settings
+	// Initially, the camera moves one turf per move. If there is no movement for
+	// cooldown_rate in deciseconds, the camera will reset to this movement rate.
+	// Every move otherwise increases sprint by acceleration_rate, until sprint
+	// exceeds sprint_threshold, and the movement rate increases by one per move.
+	// The movement rate is 1 + round(sprint / sprint_threshold).
+	var/max_sprint = 50
+	var/sprint_threshold = 20
+	var/acceleration_rate = 0.5
+	var/sprint = 10
+	var/cooldown = 0
+	var/cooldown_rate = 5
+	var/acceleration = 1
```

Comment:
Please autodoc all of these in the future. Functionality seems fine but these need to be autodoc'd
Example: `/// what this var does`